packages/travix-ui-kit/components/calendar/calendar.scss

```diff
@@ -42,8 +42,8 @@
 .ui-calendar-days__previous-month,
 .ui-calendar-days__next-month {
   align-self: center;
-  background: linear-gradient(to bottom,var(--tx-calendar-nav-button-background-color-start) 0,var(--tx-calendar-nav-button-background-color-end) 100%);
-  background: var(--tx-calendar-nav-button-background-color-end);
+  background: linear-gradient(to bottom, var(--tx-ui-button-default-normal-bg-top) 0, var(--tx-ui-button-default-normal-bg-bottom) 100%); /* W3C */
+  background: var(--tx-ui-button-default-normal-bg-top); /* Old browsers */
```

Comment:
I think these two lines need to be swapped. Look it applies the value for old browsers on my chrome.
![screenshot 2018-11-19 at 10 04 14](https://user-images.githubusercontent.com/10692413/48696560-83bae500-ebe2-11e8-9437-edb6bcdf35c5.png)